index.js

```diff
@@ -54,21 +72,16 @@ function displayBook() {
   removebtn.forEach((btn) => {
     btn.addEventListener('click', () => {
       const btnIndex = removebtn.indexOf(btn);
-      books.splice(btnIndex, 1);
+      initialBook.removeBook(btnIndex);
       displayBook();
     });
-    updateLocalStorage();
   });
+  updateLocalStorage();
 }
 displayBook();
-
 submit.addEventListener('click', () => {
   const title1 = title.value;
   const author2 = author.value;
-  books.push({
-    title: title1,
-    author: author2,
-  });
+  initialBook.addBook(title1, author2);
```

Comment:
[OPTIONAL] 

It would be user-friendly if you would create a function that will empty the input fields and trigger each time the user submits a new book.

At the moment, the input fields remain populated with the book already added to the Awesome Books.

Reply:
Thank you for your review! Much appreciated!